go.mod

```diff
@@ -252,6 +252,7 @@ require (
 	github.com/hashicorp/go-metrics v0.5.4 // indirect
 	github.com/hashicorp/go-secure-stdlib/cryptoutil v0.1.1 // indirect
 	github.com/hashicorp/go-secure-stdlib/httputil v0.1.0 // indirect
+	github.com/hashicorp/raft-snapshotagent v0.2.0 // indirect
```

Comment:
I don't think this should be included in the CE go.mod, I think it's only going to be in enterprise